### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by kwesiObuobi**
- Implement modal functionality for project details
- Show the modal when the 'See Project' button is clicked
- Hide the modal when the close button on the modal is clicked
- Generate dynamic content in the modal using javascript
- Generate HTML nodes for the `recent works` section using javascript arrays and objects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +29,50 @@
...
+                <li class="btn modal-btn">
+                    <a href="#" class="">See Live</a>
+                    <img src="assets/see-live.png" alt="see live icon">
+                </li>
+                <li class="btn modal-btn">
+                    <a href="#" class="">See Source</a>
+                    <img src="assets/modal-github.png" alt="github icon">
+                </li>
+            </ul>
+        </div> -->
```
Comment (ichala) [inline index.html:73]: - [ ] to keep your repo professional & clean , kindly remove any commented code from ur project 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Dynamic Data Loaded from array ✔️
- Good Design ✔️


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -135,6 +136,197 @@ END MOBILE NAV
...
+  width: 100vw;
+  min-height: 100%;
+  background: white;
+  z-index: 15;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  overflow: scroll;
+  gap: 1rem;
+}
```
[the remark this answers is not in the document]
Comment (kwesiObuobi) [reply style.css:161]: Hi @ichala. Thanks for flagging this. I totally missed it. 
I added the correct spaces now. 😊

File: index.html (hunk 1 above)
Comment (kwesiObuobi) [reply index.html:73]: Thanks again. Unused comments are removed now.

File: style.css [hunk 3]
```diff
@@ -135,6 +136,197 @@ END MOBILE NAV
...
+
+.modal-project-title {
+  font-size: 32px;
+}
+
+.modal-tags {
+  display: flex;
+  gap: 0.3rem;
+}
+
```
[the remark this answers is not in the document]
Comment (kwesiObuobi) [reply style.css]: Thanks, @ichala. I added some margin after the title, to match the figma design.
I think it looks fine now... @ichala 😊

**Review by BregornOriginal — APPROVED** [review]
## Hi @kwesiObuobi 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

## Highlights

✅ Linters are passing
✅ Good work making changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Comment by kwesiObuobi** [comment]
Thank you, @BregornOriginal 😃

**Pull request merged**